<launch>
	<arg name="camera" default="front_usb_cam" />
	
    <node name="uvc_camera" pkg="uvc_camera" type="uvc_camera_node" output="screen" ns="front_camera/camera">
   		<param name="camera_info_url" value="file://$(find vision)/front_usb_cam_calibration.yaml"/>
        <param name="device" value="/dev/video1" />
        <param name="width" value="320" />
        <param name="height" value="240" />
        <param name="fps" value="30" />
        <param name="topic" type="string" value="/front_camera/camera/image_raw" />
    </node>
   
</launch>